### PR TITLE
✨ Add newsletter in getting start, about and welcome notice

### PR DESCRIPTION
**Pull request by williamchong**
image
image
image

### CODE REVIEW

File: likecoin/js/admin-settings/src/pages/LikeCoinHelpPage.js [hunk 1]
```diff
@@ -40,19 +52,32 @@ function LikeCoinHelpPage() {
...
       </div>
       <div className="lcp-card">
-        <h2>{__('2. You are ready, let’s publish.', 'likecoin')}</h2>
+        <h2>{__('2. You are now ready, let’s start publishing.', 'likecoin')}</h2>
         <p>{__('Here is a video to help you understand how to publish a Writing NFT', 'likecoin')}</p>
         <iframe
           height="315"
           src="https://www.youtube.com/embed/zHmAidvifQw"
-          title="YouTube video player"
+          title={__('YouTube video player', 'likecoin')}
```
Comment (nwingt) [inline likecoin/js/admin-settings/src/pages/LikeCoinHelpPage.js:60]:
```suggestion
          title={__('Quickstart Video', 'likecoin')}
```

File: likecoin/js/admin-settings/src/pages/SponsorLikecoinPage.js [hunk 2]
```diff
@@ -108,18 +120,35 @@ function SponsorLikecoinPage() {
...
       <h2>{__('Why Web3', 'likecoin')}</h2>
       <p>{localizedWeb3Introduction}</p>
 
+      <hr />
+      <h2>{__('Subscribe to our Newsletter', 'likecoin')}</h2>
+      <p>{localizedSubscribe}</p>
+      <iframe
+          src="https://newsletter.like.co/embed"
+          width="100%"
+          height="320"
```
Comment (nwingt) [inline likecoin/js/admin-settings/src/pages/SponsorLikecoinPage.js:136]: Why is this height 320 not 150 like above?

Comment (williamchong) [reply likecoin/js/admin-settings/src/pages/SponsorLikecoinPage.js:136]: The full size widget is 320 height, with logo and title. I think it's too large and confusing to see in getting start page, so I reduced the height and let its responsiveness kick in to hide the title and logo

Comment (nwingt) [reply likecoin/js/admin-settings/src/pages/SponsorLikecoinPage.js:136]: So why not reduce it here too?

**Review by nwingt — APPROVED** [review]
I prefer to sync all Substack iframe to 150px height

File: likecoin/js/admin-settings/src/pages/SponsorLikecoinPage.js (hunk 2 above)
Comment (williamchong) [reply likecoin/js/admin-settings/src/pages/SponsorLikecoinPage.js:136]: no idea... do you think the smaller one looks better here? yes then we can reduce it here also

Comment (williamchong) [reply likecoin/js/admin-settings/src/pages/SponsorLikecoinPage.js]: reduced to 150

**Review by nwingt — APPROVED** [review]

**Pull request merged**